nengo/spa/tests/test_bind.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+
+import nengo
+from nengo import spa
+
+
+def test_basic():
+    with spa.SPA() as model:
+        model.bind = spa.Bind(dimensions=16)
+
+    inputA = model.get_module_input('bind_A')
+    inputB = model.get_module_input('bind_B')
+    output = model.get_module_output('bind')
+    # all nodes should be acquired correctly
+    assert inputA[0] is model.bind.A
+    assert inputB[0] is model.bind.B
+    assert output[0] is model.bind.output
+    # all inputs and outputs should share the same vocab
+    assert inputA[1] is inputB[1]
+    assert inputA[1].dimensions == 16
+    assert output[1].dimensions == 16
+
+
+def test_run(Simulator, seed):
+    vocab = spa.Vocabulary(16)
+    with spa.SPA(seed=seed) as model:
+        model.bind = spa.Bind(dimensions=32, vocab=vocab)
```

Comment:
How can you assign a 16-dimensional vocab to something that requires 32 dimensions? Shouldn't this raise an error?